### ISSUE TITLE
[Immersive Engineering] Bullet Chicken outputs weird unlocalised bullet type that doesn't seem to be used by IE (possibly NBT related?)

### ISSUE THREAD

**Issue by Overato** [GenDeathrow/MoreChickens#30]
Suggested fix is to either implement multiple bullet types for IE crossmod chicken or make it lay bullet casings instead.

**Comment by GenDeathrow** [comment]
Strange was working fine for me. I will check into it again. Thanks for the report. The main reason that chicken is there atm is to prevent old worlds from crashing cause of an id change.

**Comment by GenDeathrow** [comment]
yeah, looks like he changed his bullets. I apologize I didn't' get to this sooner. but trying to get some updates out. instead of meta data he is using nbt data. I'll get this fixed as soon as I can

**Comment by IAmVensey** [comment]
Hello,

Has there been any update to this issue?

**Comment by MatthewLickers** [comment]
No. The laid bullets still don’t work
